SLURM_JOB_ID: 64208878
SLURM_JOB_USER: vsc37097
SLURM_JOB_ACCOUNT: lp-emedia
SLURM_JOB_NAME: project_h100.slurm
SLURM_CLUSTER_NAME: wice
SLURM_JOB_PARTITION: gpu_h100
SLURM_NNODES: 1
SLURM_NODELIST: s16g15
SLURM_JOB_CPUS_PER_NODE: 16
SLURM_JOB_GPUS: 0,1
Date: Mon Mar 31 13:29:34 CEST 2025
Walltime: 00-08:00:00
========================================================================
W0331 13:29:40.600000 3669153 /vsc-hard-mounts/leuven-data/370/vsc37097/miniconda3/envs/torch/lib/python3.10/site-packages/torch/distributed/run.py:793] 
W0331 13:29:40.600000 3669153 /vsc-hard-mounts/leuven-data/370/vsc37097/miniconda3/envs/torch/lib/python3.10/site-packages/torch/distributed/run.py:793] *****************************************
W0331 13:29:40.600000 3669153 /vsc-hard-mounts/leuven-data/370/vsc37097/miniconda3/envs/torch/lib/python3.10/site-packages/torch/distributed/run.py:793] Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
W0331 13:29:40.600000 3669153 /vsc-hard-mounts/leuven-data/370/vsc37097/miniconda3/envs/torch/lib/python3.10/site-packages/torch/distributed/run.py:793] *****************************************
Traceback (most recent call last):
  File "/vsc-hard-mounts/leuven-data/370/vsc37097/thesis/train_hpc.py", line 29, in <module>
    from components.augmentation import (
  File "/vsc-hard-mounts/leuven-data/370/vsc37097/thesis/components/augmentation.py", line 3, in <module>
    from pre_processing import hand_mirroring
ModuleNotFoundError: No module named 'pre_processing'
Traceback (most recent call last):
  File "/vsc-hard-mounts/leuven-data/370/vsc37097/thesis/train_hpc.py", line 29, in <module>
    from components.augmentation import (
  File "/vsc-hard-mounts/leuven-data/370/vsc37097/thesis/components/augmentation.py", line 3, in <module>
    from pre_processing import hand_mirroring
ModuleNotFoundError: No module named 'pre_processing'
E0331 13:29:43.317000 3669153 /vsc-hard-mounts/leuven-data/370/vsc37097/miniconda3/envs/torch/lib/python3.10/site-packages/torch/distributed/elastic/multiprocessing/api.py:869] failed (exitcode: 1) local_rank: 0 (pid: 3669173) of binary: /data/leuven/370/vsc37097/miniconda3/envs/torch/bin/python
Traceback (most recent call last):
  File "/data/leuven/370/vsc37097/miniconda3/envs/torch/bin/torchrun", line 8, in <module>
    sys.exit(main())
  File "/data/leuven/370/vsc37097/miniconda3/envs/torch/lib/python3.10/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 355, in wrapper
    return f(*args, **kwargs)
  File "/data/leuven/370/vsc37097/miniconda3/envs/torch/lib/python3.10/site-packages/torch/distributed/run.py", line 919, in main
    run(args)
  File "/data/leuven/370/vsc37097/miniconda3/envs/torch/lib/python3.10/site-packages/torch/distributed/run.py", line 910, in run
    elastic_launch(
  File "/data/leuven/370/vsc37097/miniconda3/envs/torch/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 138, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/data/leuven/370/vsc37097/miniconda3/envs/torch/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 269, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
============================================================
train_hpc.py FAILED
------------------------------------------------------------
Failures:
[1]:
  time      : 2025-03-31_13:29:43
  host      : s16g15.tier2.hpc.kuleuven.be
  rank      : 1 (local_rank: 1)
  exitcode  : 1 (pid: 3669174)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
------------------------------------------------------------
Root Cause (first observed failure):
[0]:
  time      : 2025-03-31_13:29:43
  host      : s16g15.tier2.hpc.kuleuven.be
  rank      : 0 (local_rank: 0)
  exitcode  : 1 (pid: 3669173)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
============================================================
